pkcs11/src/new/functions/object_management.rs

```diff
@@ -13,7 +13,7 @@ const MAX_OBJECT_COUNT: usize = 10;
 impl Pkcs11 {
     pub fn find_objects(
         &self,
-        session: &mut Session,
+        session: &Session,
```

Comment:
Removed that for simplicity. Will check in a later step what is our strategy against data races and how the signatures should change depending on that